Automattic/wp-calypso
# Settings: Use new language picker in Calypso

#### Changes proposed in this Pull Request

* Use the new language picker in Calypso's language pickers (site language and UI/account language)

#### Testing instructions



* Ensure the language pickers at `/settings/general/:site` and `/me/account` work as expected both on desktop and mobile. Ensure that searching behaves as expected and that selecting a new language works.
* Ensure that checking the empathy mode and fallback checkboxes is working and that they appear appropriately on mobile and desktop (these are only present on the `/me/account` language picker).
